geometry/media_r3b.geo

```diff
@@ -9,21 +9,12 @@ vacuum             1  1.e-16  1.e-16  1.e-16
 carbon             1  12.011  6.0  2.265
                    0  1  20.  .001
                    0
-carboncooling      1  12.011  6.0  1.60
```

Comment:
These materials are needed for other detectors, please, add all of them again

Reply:
I put it back, I can not remember doing this myself, it probably happened during my rebase.